aws/resource_aws_instance_test.go

```diff
@@ -2814,6 +2817,12 @@ resource "aws_instance" "test" {
   instance_type   = "m1.small"
   security_groups = ["${aws_security_group.tf_test_test.name}"]
   user_data       = "foo:-with-character's"
+
+  metadata_options {
```

Comment:
Please create a new test function and test configuration so we can verify existing testing for regressions.